src/coco/zeplinComponent/util/zeplinComponentUtil.ts

```diff
@@ -4,24 +4,27 @@ import BarrelDetailsResponse from "../../../common/domain/barrel/model/BarrelDet
 
 function getComponentsFromSections(barrel: BarrelDetailsResponse): ZeplinComponent[] {
     return getComponentsFromSectionsAdditive(
+        barrel._id,
         barrel.name,
         barrel.componentSections.map(section => ({
             section,
-            breadcrumbs: []
+            nameBreadcrumbs: [],
+            idBreadcrumbs: []
```

Comment:
How about having a single property `breadcrumbs` like before, but this time with array of objects and each object has id as the key and name as the value?

Reply:
I didn't want to make changes in connected components related parts for this change. I still may do it if you think it would be better

Reply:
This is okay, that's just an improvement you can do later.